### PR TITLE
A201 subcostcode

### PR DESCRIPTION
**Pull request by harrietc52**

### CODE REVIEW

**Comment by NikolausC** [comment]
Can't access Jenkins remotely, but if it's complaining about the database password being wrong, it may need switching in the build config, then run the tests again

File: spec/models/node_spec.rb [hunk 1]
```diff
@@ -37,7 +39,56 @@
     expect(root).not_to have_attribute('owner')
   end
 
+  describe '#scopes' do
```
[the remark this answers is not in the document]
Comment (harrietc52) [reply spec/models/node_spec.rb]: Yep :) think we have covered those tests elsewhere

**Comment by harrietc52** [comment]
Where is the build config?

**Comment by harrietc52** [comment]
Not sure I'm getting access to the interal network this morning, so might not be able to change the Jenkins config?

**Comment by emrojo** [comment]
retest this please

**Comment by emrojo** [comment]
retest this please

**Comment by emrojo** [comment]
Ok, the password in jenkins is changed, so now it should stop giving errors.

**Comment by harrietc52** [comment]
Dont merge yet :) still drag and drop etc to sort

**Pull request merged**